esp_wrover_kit/idf_component.yml

```diff
@@ -1,15 +1,20 @@
-version: "1.4.3"
+version: "1.5.0"
 description: Board Support Package for ESP-WROVER-KIT
 url: https://github.com/espressif/esp-bsp/tree/master/esp_wrover_kit
 
 targets:
     - esp32
 
 dependencies:
-  idf: ">=4.4" # esp_lcd component was added in v4.4
+  idf: ">=4.4.5"
```

Comment:
I'm sorry, but can you please remind me why do we need 4.4.5?

Reply:
We need changes in `esp_lcd` (registering flush callback) for `esp_lvgl_port`. This feature was backported and released in version 4.4.5